src/TraceEvent/Computers/TraceManagedProcess.cs

```diff
@@ -3586,6 +3638,13 @@ public struct InliningFailureResult
         public string Reason;
     }
 
+    public enum CompilationThreadKind
```

Comment:
If I recall correctly, we intentionally moved all enums to the top level namespace.

Reply:
Looking at the file, we defiantly have some enums in Microsoft.Diagnostics.Tracing.Analysis.GC.   This is currently in Microsoft.Diagnostics.Tracing.Analysis.JIT which I think is fine.     So I think it is OK to leave it.